src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  &--4 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+
+    &:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
And with another modificator it should show different count of active stars
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/f797f2e4-c7eb-4f8f-b298-534124ca4ab5)